setup.py

```diff
@@ -236,10 +236,13 @@ def prepare_env_for_osx():
             'Operating System :: MacOS',
             'Intended Audience :: Science/Research',
             'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+            'Programming Language :: Python :: 2.6',
```

Comment:
I have intention to drop 2.6, 2.7, 3.4, 3.5 and only keep 2.7 and >= 3.7 so people can move one.

Reply:
In my own packages I try to support the currently supported Python versions. If you don't mind, I think it's better to remove 3.5 and 3.6 only when these versions are also EOL, or this package contains code that is not compatible with these versions (use of f-strings, for example). There's no reason to deny people to use this package on 3.5 if it still works there.

I have remove the the EOL versions now.

Reply:
ok, it's fine with me. So just keep python 2.7 (phenix package is still using python 2.7) and >=3.5